packages/inference/src/index.ts

```diff
@@ -3,5 +3,6 @@ export * from "./errors.js";
 export * from "./types.js";
 export * from "./tasks/index.js";
 import * as snippets from "./snippets/index.js";
+export * from "./lib/makeRequestOptions.js"
```

Comment:
```suggestion
export * from "./lib/getProviderHelper.js";
export * from "./lib/makeRequestOptions.js";
```

you will also need `getProviderHelper` utility (+ fix lint)